docs/controls/Gantt.md

```diff
@@ -202,17 +202,19 @@ ganttInstance.loadContent(dataArray);
-| showActionLegend | boolean  | true                                           | Toggle to show graph legend                                                          |
-| padding          | object   | `{ top: 10, bottom: 5, left: 100, right: 50 }` | Refer [Padding](Padding.md)                                                          |
-| clickPassThrough | object   | null                                           | Refer [Pass Through](#Pass-Through)                                                  |
-| pan              | object   | {}                                             | Refer [Panning](./Panning.md)                                                        |
+| Property Name          | Expected | Default                                         | Description                                                                          |
+| ---------------------- | -------- | ----------------------------------------------- | ------------------------------------------------------------------------------------ |
+| bindLegendTo           | string   | null                                            | If DOM id provided, binds legend into that container (Example: `"#legendContainer"`) |
+| locale                 | object   | LOCALE.en_US                                    | Locale object for X-Axis tick values                                                 |
+| throttle               | number   | (1000/60) => time in ms                         | Delay between resize of a browser window                                             |
+| dateline               | array    | []                                              | Refer [Dateline](../helpers/Dateline.md)                                             |
+| actionLegend           | array    | []                                              | Refer [Action Legend](#action-legend)                                                |
+| showActionLegend       | boolean  | true                                            | Toggle to show graph legend                                                          |
+| padding                | object   | `{ top: 10, bottom: 10, left: 100, right: 50 }` | Refer [Padding](Padding.md)                                                          |
+| clickPassThrough       | object   | null                                            | Refer [Pass Through](#Pass-Through)                                                  |
+| pan                    | object   | {}                                              | Refer [Panning](./Panning.md)                                                        |
+| removeContainerPadding | boolean  | false                                           | Toggle to remove carbon container top and bottom padding                             |
+| legendPadding          | object   | `{ top: 4, bottom: 4, left: 8, right: 8 }`      | Sets the legend padding and removes the carbon default margin of 8px around legend   |
```

Comment:
Nit: since you're already making changes in the table, I would alphabetize the properties.

Reply:
Alphabetized the props [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/751bb4e91a16e2653e7d094c18ae089e4393678f).